app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -2,15 +2,66 @@
   <div class="container">
     <div class="row">
       <h2>Sessions</h2>
-      <div> Coming soon - Sessions list!</div>
+    </div>
+    <template v-if="sessionsMetadata">
+      <div class="row fw-bold py-2">
+        <div class="col-3 session-col-header">Saved</div>
+        <div class="col-2 session-col-header">Label</div>
+        <div class="col-2 text-center session-col-header">Load</div>
+      </div>
+      <div class="row py-2" v-for="session in sessionsMetadata" :key="session.id">
```

Comment:
The empty sessions page might be nicer with some placeholder ("no saved sessions yet") than the table with no rows?

![image](https://user-images.githubusercontent.com/1558093/190201115-f5430b2c-1d35-4a8c-8c3f-a9bfb5f8a971.png)